src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx

```diff
@@ -81,3 +84,27 @@ export const FundingTargetPage: React.FC = () => {
     </Form>
   )
 }
+
+const FormattedNumberInputWrapper: React.FC<
+  FormItemInput<CurrencySelectInputValue>
+> = props => {
+  const currency = props.value?.currency ?? 'eth'
+  return (
+    <FormattedNumberInput
+      value={props.value?.amount}
+      onChange={amount => props.onChange?.({ amount, currency })}
+      accessory={
+        <InputAccessoryButton
+          withArrow
+          content={currency === 'eth' ? 'ETH' : 'USD'}
```

Comment:
perhaps we could use the currency constants here and elsewhere: `constants/currency.ts`

Reply:
I think we need a lot of standardisation on our code in general 😅